source/release-notes/4-2-x-breaking-changes.rst

```diff
@@ -19,6 +19,28 @@ before upgrading. Please review the following and apply changes as required:
 
 |hr|
 
+------------------------------------
+Breaking Changes in CrafterCMS 4.2.2
+------------------------------------
+CrafterCMS uses TinyMCE version 7 for the RTE control.
+The following properties were removed from the RTE control in CrafterCMS version 4.2.2:
+
+- ``forceRootBlockPTag``
+
+  This property uses the **forced_root_block** option in TinyMCE which has changed in
+  `version 6 <https://www.tiny.cloud/docs/tinymce/5/6.0-upcoming-changes/#options>`__. Since the option has changed,
+  the property is no longer needed.
+- ``forceBRTags``
```

Comment:
As above, let's use the label.

Overall the relevant thing here would be to explain what should be done if these things were in use before. This one seems covered by the blurb below, but the above one feels incomplete.

For both, we should mention that now to customise these behaviours, one would use the TinyMCE config in ui.xml